bin/test-external-partner-project.js

```diff
@@ -113,19 +86,32 @@ try {
     debug('Skipping Smoke Test');
   } else {
     debug('Running Smoke Test');
+    try {
+      execExternal(`${useYarn ? 'yarn install' : 'npm install'}`);
```

Comment:
How come we first install and then modify the package. Json?

Reply:
For the smoke-test we use their setup first